Fray c/Program.cs

```diff
@@ -0,0 +1,222 @@
+            invoiceList.Add(invoiceTemp4);
+            invoiceList.Add(invoiceTemp5);
+
+            return invoiceList;
+        }
+        static void Main(string[] args)
+        {
+            List<Customer> customerList = new List<Customer>();
+            ProductList productList = new ProductList();
+            List<Product> productMinStockList = new List<Product>();
+            List<Invoice> invoiceList = new List<Invoice>();
+            
+            List<Customer> customer55List = new List<Customer>();
+            List<Customer> customerBirthDayList = new List<Customer>();
+
+            customerList = CustomerList();
+            productList = ProductList();
+            invoiceList = InvoiceList();
+
+            Seller sellerTemp = new Seller();
+            sellerTemp.SellerId = 1;
+            sellerTemp.Name = "Carlos Bodger";
+
+            float SalesTotal = 0;
+            foreach(var invoice in invoiceList)
+            {
+                Customer customer55 = new Customer();
+                Customer customerBirthDay = new Customer();
+                int customerId = invoice.CustomerId;
+
+                SalesTotal+= invoice.ProductPrice*invoice.ProductQuantity;
+
+                customer55 = customerList.Find(x => x.CustomerId == customerId && x.Age > 55);
+                customerBirthDay = customerList.Find(x => x.CustomerId == customerId && x.BirthDate.Month == DateTime.Now.Month);
+                
+                if(customer55 != null && !customer55List.Contains(customer55))
+                    customer55List.Add(customer55);
+                
+                if(customerBirthDay != null && !customerBirthDayList.Contains(customerBirthDay))
+                    customerBirthDayList.Add(customerBirthDay);
+            }
+
+            for(int i=0; i< productList.ProductsList.Count; i++)
+            {
+                if(productList.ProductsList[i].Stock <= productList.MinStock)
+                {
+                    Product productMinStock = new Product();
+                    productMinStock = productList.ProductsList[i];
+                    productMinStockList.Add(productMinStock);
+                }
+            }
+
+
+            Console.WriteLine("Total Ventas: " + SalesTotal);
+            Console.WriteLine("");
+
+            Console.WriteLine("Clientes mayores a 55 anios: ");
+            foreach(var customer in customer55List)
+            {
+                Console.WriteLine($"Nombres: {customer.Name}, Anios: {customer.Age}");
+            }
+            
+            Console.WriteLine("");
+
+            Console.WriteLine("Clientes que cumplan anios este mes: ");
+            foreach(var customer in customerBirthDayList)
+            {
+                Console.WriteLine($"Nombres: {customer.Name}, Anios: {customer.Age}");
+            }
+            
+            Console.WriteLine("");
+            
+            Console.WriteLine("Productos con stock que estan en el minimo o por debajo: ");

```

Comment:
only english.
our customers are foreigner so they don't going to understand our language.